tests/ui/parser/issues/issue-104367.stderr

```diff
@@ -1,11 +1,3 @@
-error: mismatched closing delimiter: `)`
-  --> $DIR/issue-104367.rs:5:15
-   |
-LL |             #![w,)
-   |               ^  ^ mismatched closing delimiter
-   |               |
-   |               unclosed delimiter
```

Comment:
this is a typical mismatched one, 
the right fix maybe `[w,]` or `(w,)`, 
it depends on the specific scenario.
but the new current hint is suggesting the second fix or adding another extra `(`?
seems the old diagnostics is better.